lib/active_admin_import/model.rb

```diff
@@ -132,6 +131,11 @@ def define_methods_for(attr_name)
       end
     end
 
+    def encode(data)
+      data.encode(force_encoding, 'binary',
+                  invalid: :replace, undef: :replace,  replace: '').
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.